raspberryPI3/ros2_ws/src/can/src/can_rx_node.cpp

```diff
@@ -219,7 +219,7 @@ class can_rx : public rclcpp::Node {
                 auto navsat_msg = sensor_msgs::msg::NavSatFix();
                 bool use_gps_shift_fix = this->get_parameter("gps_shift_fix").as_bool();
                 navsat_msg.header.stamp = this->get_clock()->now();
-                navsat_msg.header.frame_id = "gps";
+                navsat_msg.header.frame_id = "base_link";

```

Comment:
base_link standard name used to test the gps or the imu

Reply:
`base_link` is the standard frame for the base of a robot/car (chassis). In our case, we do not use much transforms and frames so we can use `base_link`.
When building a complex robot, such as Tiago, you must define links (parts) and joints (connection between parts). There are tools that make the robot kinematics easy if these are well defined.